pf9-autodeploy.yml

```diff
@@ -1,22 +1,40 @@
 ---
-- hosts: hypervisors-ubuntu
+- hosts:
+    - hypervisors
+    - containervisors
   become: true
   gather_facts: False
   pre_tasks:
     - name: install python2
-      raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
+      raw: if [ -e /etc/lsb-release -a ! -e /usr/bin/python ]; then (apt -y update && apt install -y python-minimal); fi
 
 - hosts: hypervisors
   become: true
+  vars_files:
+    - pf9-role-metadata.yml
   roles:
-   - common
-   - ntp
-   - pf9-auth
-   - pf9-hostagent
-   - neutron-prerequisites
-   - secure-live-migration
+    - common
+    - ntp
+    - qemu-kvm-ev
```

Comment:
The `os_family` conditionals which are present on the tasks in this role should be placed here instead.